distributionviewer/core/static/js/app/components/containers/chart-config-container.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import ChartConfig from '../views/chart-config';
+import * as metricApi from '../../api/metric-api';
+
+
+class ChartConfigContainer extends React.Component {
+  componentDidMount() {
+    metricApi.getMetrics();
+  }
+
+  handleSubmit(e) {
```

Comment:
By convention private methods start with an underscore, like `_handleSubmit()`. I'll admit I haven't been consistent about this myself.